python/samples/documentation_examples/configuring_prompts.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) Microsoft. All rights reserved.
+import asyncio
+
+from service_configurator import add_ai_service
+
+import semantic_kernel as sk
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+from semantic_kernel.contents.chat_history import ChatHistory
+from semantic_kernel.core_plugins import ConversationSummaryPlugin
+from semantic_kernel.prompt_template.input_variable import InputVariable
+from semantic_kernel.prompt_template.prompt_template_config import PromptTemplateConfig
+
+
+async def main():
+    # Initialize the kernel
+    kernel = sk.Kernel()
+
+    # Add the service to the kernel
+    # use_chat: True to use chat completion, False to use text completion
+    kernel = add_ai_service(kernel=kernel, use_chat=True)
+
+    service_id = "default"
+    execution_settings = PromptExecutionSettings(
+        service_id=service_id, max_tokens=ConversationSummaryPlugin._max_tokens, temperature=0.1, top_p=0.5
+    )
+    prompt_template_config = PromptTemplateConfig(
+        template=ConversationSummaryPlugin._summarize_conversation_prompt_template,
+        description="Given a section of a conversation transcript, summarize the part of" " the conversation.",
+        execution_settings=execution_settings,
+    )
+
+    # Import the ConversationSummaryPlugin
+    kernel.import_plugin_from_object(
+        ConversationSummaryPlugin(kernel=kernel, prompt_template_config=prompt_template_config),
+        plugin_name="ConversationSummaryPlugin",
+    )
+
+    # Create the history
+    history = ChatHistory()
+
+    # Create the prompt with the ConversationSummaryPlugin
+    prompt = """{{ConversationSummaryPlugin.SummarizeConversation $history}}
+    User: {{$request}}
+    Assistant:  """
+
+    req_settings = kernel.get_service("default").get_prompt_execution_settings_class()(service_id=service_id)
```

Comment:
this can be done with: `kernel.get_service("default").instantiate_prompt_execution_settings(service_id=service_id)`